app/models/CountryList.scala

```diff
@@ -29,5 +30,21 @@ case class CountryList(countries: Seq[Country]) {
 }
 
 object CountryList {
+
+  val customReads: Reads[CountryList] = Reads[CountryList] {
+    case JsArray(value) =>
+      JsSuccess(
+        CountryList(
+          value.map(
+            entry => (entry \ "countryOfRouting").as[Country]
+          )
+        )
+      )
+    case _ => JsError("error")
```

Comment:
I left this generic error message here but we might wan to change this to include a more meaningful error? 

I.e. `<class>::<method> - Failed to read country list from cache`